demos/blockfactory/factory_utils.js

```diff
@@ -902,9 +902,9 @@ FactoryUtils.defineAndGetBlockTypes = function(blockDefsString, format) {
 FactoryUtils.injectCode = function(code, id) {
   var pre = document.getElementById(id);
   pre.textContent = code;
-  code = pre.textContent;
-  code = PR.prettyPrintOne(code, 'js');
-  pre.innerHTML = code;
+  // Remove the 'prettyprinted' class, so that Prettify will recalculate.
+  pre.className = pre.className.replace('prettyprinted', '');
```

Comment:
I don't think this is sufficient. If I name it 'prettyprettyprintedprinted' the result of the replace is 'prettyprinted'

Reply:
This is not a user-definable field.  The class is hardcoded in our HTML, and the pretty printer appends 'prettyprinted' when the text is syntax highlighted.

Reply:
Got it. I was trying to figure out how the injection worked but was unclear where the input came from. =)